snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -97,10 +97,19 @@ public class Snapshot: NSObject {
         print("snapshot: \(name)") // more information about this, check out https://github.com/fastlane/fastlane/tree/master/snapshot
 
         sleep(1) // Waiting for the animation to be finished (kind of)
-        XCUIDevice.sharedDevice().orientation = .Unknown
+
+        #if os(tvOS)
+            XCUIApplication().childrenMatchingType(.Browser).count
```

Comment:
Would you mind adding this as a comment to the code base here for future reference? Also mention how it's important to find a gesture that's not used elsewhere in UI Tests, as it will be fetched from the `xcodebuild` output